typings/compileSolidity.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'compileSolidity' {
+    interface CompileContractsOutput {
+        output?: string;
+        error?: string[];
+    }
+}
```

Comment:
Generally speaking, `.d.ts` files are only used for defining types of JavaScript modules.  In this case, you can just put the `interface CompileContractsOutput { ... }` at the top of your script's source file, or in another file in `source/libraries` if that makes more sense to you.  Regular `.ts` files are generally much easier to work with.

If you decide to go with a separate file for this (note: I'm not implying that is the right solution, I leave that to you) then the file would look something like:
```
interface CompileContractsOutput {
    output?: string;
    error?: string[];
}
```
And then in any other file in your project you would do:
```
import { CompileContractsOutput } from './CompileContractsOutput.ts'; // path is relative to directory this file is in
...
```

Reply:
Ok, good to know.  Thanks for the clarification.  I'll move the interface to the `.ts` file, since that's probably the only place it will be used.